src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,77 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  justify-content: center;
+  padding: 35px;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  display: flex;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  margin-right: 20px;
+  width: 340px;
+}
+
+.formInput {
+  display: flex;
+  flex-direction: row;
+}
+
+.buttonWrapper {
+  margin-left: 38px;
```

Comment:
This is really confusing me. We shouldn't allow the input text element to be larger than its containing parent in the first place. It's even worse to attempt to compensate for it by forcing a margin on the button element.

Reply:
How should we solve this problem? The padding of the input box creates this bug.

Reply:
Took it out